src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/config/Config.java

```diff
@@ -404,6 +442,44 @@ public void setValues(JSONObject formData) {
         replicationConfig = ReplicationConfig.createReplicationConfigFromJSON(formData);
     }
 
+    private void initializeGerritCommands() {
+        List<String> stringLabels = new ArrayList<>(Arrays.asList("verified", "code-review"));
+
+        for (VerdictCategory label : categories) {
+            stringLabels.add(label.getVerdictValue().toLowerCase());
+
+            if (!label.getVerdictValue().equalsIgnoreCase("Code-Review") && !label.getVerdictValue().equalsIgnoreCase("Verified")) {
```

Comment:
[minor] since they are used many times, what about extracting the strings "Code-Review" and "Verified" as constants?

Reply:
I've extracted "Code-Review" & "Verified" labels in Config.java and replaced all their occurrences /w Config.CODE_REVIEW / Config.VERIFIED. Let me know if that's not ok.